testgrid/cmd/configurator/config_test.go

```diff
@@ -479,3 +481,20 @@ func TestReleaseBlockingJobsMustHaveTestgridDescriptions(t *testing.T) {
 		}
 	}
 }
+
+func TestNoEmpyMailToAddresses(t *testing.T) {
+	for _, dashboard := range cfg.Dashboards {
+		for _, dashboardtab := range dashboard.DashboardTab {
+			intro := fmt.Sprintf("dashboard_tab %v/%v", dashboard.Name, dashboardtab.Name)
+			if dashboardtab.AlertOptions != nil {
+				mails := strings.Split(dashboardtab.AlertOptions.AlertMailToAddresses, ",")
```

Comment:
Consider trimming leading/trailing whitespace as well.

Reply:
`mail` is happy with leading/trailing whitespace, do you want this test to fail if there is leading/trailing whitespace?

Reply:
Hmm. Welp this is handled already by TestGrid code when the addresses are used, so either way.